code/style.css

```diff
@@ -1,4 +1,257 @@
-html {
-  background: #0025ff;
-  color: #fff;
+
+body {
+  font-family: 'Sora', sans-serif;
+  margin: 0 auto;
+  padding: 0;
+  min-height: 100vh;
+  max-width: 100%;
+  background-color:whitesmoke;
+  flex-direction: column;
+  justify-content: center;
 }
+header{
+  width:100%;
+  background:white;
+  padding: 20px;
+  position: relative;
+  top: 0;
+  display: flex;
+  align-items: center;
+  box-sizing: border-box;
+}
+.footer{
+  background-color: blueviolet;
+  height: 50px;
+  padding-top: 15px;
+  display: flex;
+  justify-content: center;
+  color: white;
+}
+img {
+  max-width: 100%;
+  height: auto;
+  display: block;
+}
+*,
+*::before,
+*::after {
+  box-sizing: inherit;
+  margin: 0px;
+  padding: 0px;
+}
+.read-more:hover{
+  background-color: rgb(137, 75, 196);
+}
+.read-more{
+  display: block;
+  padding: 12px 28px;
+  margin: auto;
+  border-radius: 5px;
+  border:steelblue;
+  cursor: poiner; F
```

Comment:
This capital "F" that has sneaked its way into the code causes some error alerts in VS Code.